src/main/java/layers/FinishLayer.java

```diff
@@ -76,12 +108,21 @@ public FinishLayer(int score, boolean hasWon) {
      */
     private void select() {
         switch(selected) {
-            case 0: addLayer(new GameLayer()); // Start new game
+            case 0: _HighScoresHandler.addScore(playerName, score);
+                    addLayer(new GameLayer()); // Start new game
                     removeLayer();
                     break;
-            case 1: addLayer(new TitleLayer()); // Title Screen
+            case 1: _HighScoresHandler.addScore(playerName, score);
+                    addLayer(new TitleLayer()); // Title Screen
                     removeLayer();
                     break;
+            case 2:
+            case 3:
+            case 4:
+            case 5:
+            case 6:
```

Comment:
The usage of an case without a "break" is that he will continue to the next case and execute that one.

Here we have the problem that if selected is 2/3/4/5/6, then we will want to execute the same method.

This is not really beautiful, but it is better then placing the method call 5 times!
<review